lib/src/request_processor.cc

```diff
@@ -217,8 +217,7 @@ RequestProcessor::~RequestProcessor()
 void RequestProcessor::processRequest(const RequestType& request_type,
                                       const PCPClient::ParsedChunks& parsed_chunks)
 {
-    LOG_TRACE("About to validate and process PXP request message: {1}",
-              parsed_chunks.toString());
+    LOG_TRACE("About to validate and process PXP request message");
```

Comment:
Not sure I'd be concerned about logging sensitive stuff at trace. I don't think the remaining message is very valuable on its own, so I'd lean towards removing it altogether.